aimmo-game/simulation/direction.py

```diff
@@ -16,6 +24,13 @@ def dict(self):
     def __repr__(self):
         return 'Direction(x={}, y={})'.format(self.x, self.y)
 
+    @staticmethod
+    def from_dict(other):
+        try:
+            return Direction(**other.dict)
```

Comment:
Looks like this is now unnecessary as `from_dict` is only called with actual dicts (which makes sense with the new name).